### ISSUE TITLE
[Feature Request] blocking images

### ISSUE THREAD

**Issue by LukasThreiro** [jangernert/FeedReader#857]
I would like to be able to block images in articles. This is annoying when the picture takes half the space in the article. I do not need it.

**Comment by brendanlong** [comment]
You want an option to disable all pictures?

**Comment by LukasThreiro** [comment]
Exactly. Pictures do not bring any substantive content, but only distract me.
